tests/python_tests/test_continuous_batching.py

```diff
@@ -158,6 +158,45 @@ def test_chat_scenario_vs_stateful(model_id, generation_config_kwargs: Dict, pip
     # Test that finish_chat() doesn't fail just in case.
     cb_pipe.finish_chat()
 
+
+generation_configs = [
+    dict(do_sample=False, max_new_tokens=20),
+    dict(do_sample=False, num_beam_groups=3, num_beams=15, num_return_sequences=1, max_new_tokens=10, diversity_penalty=1.0, repetition_penalty=1.0)
+]
+questions = [
+    '1+1=',
+    'What is the previous answer?',
+    'Why is the Sun yellow?',
+    'What was my first question?'
+]
+@pytest.mark.parametrize("generation_config_kwargs", generation_configs[1:])
+@pytest.mark.parametrize("model_id", get_chat_models_list())
+@pytest.mark.parametrize("pipeline_type", [PipelineType.PAGED_ATTENTION, PipelineType.PROMPT_LOOKUP_DECODING, PipelineType.SPECULATIVE_DECODING] )
+@pytest.mark.precommit
+def test_continuous_batching_add_request_health_check(model_id, generation_config_kwargs: Dict, pipeline_type):
+    _, _, models_path = download_and_convert_model(model_id)
```

Comment:
We can use `generate_and_compare` instead https://github.com/openvinotoolkit/openvino.genai/blob/9fb52b6bf553d7286ed75db59dc9407ffa3be87c/tests/python_tests/utils/ov_genai_pipelines.py#L205